stl/src/locale0.cpp

```diff
@@ -43,6 +43,19 @@ struct _Fac_node { // node for lazy facet recording
     void operator delete(void* _Ptr) noexcept { // replace operator delete
         _free_dbg(_Ptr, _CRT_BLOCK);
     }
+#else

```

Comment:
When `_DEBUG` isn't defined, `_malloc_dbg` and `_free_dbg` turn into calls to `malloc` and `free`, so this entire change could be accomplished by simply removing the `#ifdef _DEBUG`.